hubblestack/utils/signing.py

```diff
@@ -194,32 +256,48 @@ def __init__(self, public_crt, ca_crt):
         self.trusted = list()
         # NOTE: trusted is mostly useless. We do use it in
         # testing, and that's probably about it
-
+        seconds_day = 86400
         already = set()
         for i in read_certs(ca_crt):
+            log_level = log.debug
             digest = i.digest('sha1')
             if digest in already:
                 continue
             already.add(digest)
             digest = digest.decode() + " " + stringify_ossl_cert(i)
-            log.debug('adding %s as a trusted certificate approver', digest)
             self.store.add_cert(i)
             self.trusted.append(digest)
+            log_level = log.debug
+            if check_verif_timestamp(digest, dampener_limit=seconds_day) == True:
```

Comment:
Redundant comparison to True, check_verif_timestamp() returns bool value already.